Story/StoryHooks.cs

```diff
@@ -102,19 +156,19 @@ private void SLOracleSwarmer_BitByPlayer(On.SLOracleSwarmer.orig_BitByPlayer ori
         private void OracleSwarmer_BitByPlayer(On.OracleSwarmer.orig_BitByPlayer orig, OracleSwarmer self, Creature.Grasp grasp, bool eu)
         {
             orig(self, grasp, eu);
-            if (self.slatedForDeletetion == true) 
+            if (self.slatedForDeletetion == true)
             {
                 SwarmerEaten();
             }
         }
 
-        private void SwarmerEaten() 
+        private void SwarmerEaten()
```

Comment:
Next time do these formatting changes in the different PR. It makes reading diffs harder

Reply:
Yeah it happens from me hitting shift+ctrl then k to autoformat  the current document.